apps/frontend/src/app/components/level-indicator/level-indicator.component.html

```diff
@@ -0,0 +1,13 @@
+<div
+  [ngStyle]="{ 'background-color': levelColor }"
+  [ngClass]="[prestige ? 'pl-1 pr-3' : 'px-2', textColor]"
+  class="flex gap-1 rounded py-0 font-display text-xl shadow"
```

Comment:
How does bold text here look? IIRC that's what we do ingame, and looks a little thin in your screenshots.

Reply:
Using bold looks like this: 
![image](https://github.com/user-attachments/assets/1bad39c3-6659-4022-b644-bc7db78858f1)

As ingame looks like this:
![image](https://github.com/user-attachments/assets/9264edc3-53d0-477a-9a32-fb0085d86e58)

Looks more closer to ingame using bold, but font family is still different I think.

Reply:
Added b tag to level number.